goss-testing/tests/ncn/goss-ceph-status.yaml

```diff
@@ -0,0 +1,50 @@
+#
+# MIT License
+#
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+{{ $thisnode := .Vars.thisnode }}
+{{ $logrun := .Env.GOSS_BASE | printf "%s/scripts/log_run.sh" }}
+package:
+    # This package is installed on all NCNs
+    ceph-common:
+        title: ceph-common package
+        meta:
+            desc: Check that Ceph is installed
+            sev: 0
+        installed: true
+command:
+    # We expect "ceph -s" to work on the first three storage nodes
+    # and any master node
+    {{if $thisnode | regexMatch "^ncn-(m|s00[1-3])" }}
+        {{ $testlabel := "ceph_s_runs" }}
+        {{$testlabel}}:
+            title: ceph -s
+            meta:
+                desc: Check that ceph -s successfully executes
+                sev: 0
+            exec: |-
+                "{{$logrun}}" -l "{{$testlabel}}" \
+                    /usr/bin/ceph -s
```

Comment:
```suggestion
                    /usr/bin/ceph health
```

Reply:
it is a little cleaner and only gives the HEALTH_OK/WARN/.....

Reply:
Since the output is only seen if the test failure needs to be debugged, it seems like it would be more helpful to have it show the additional information. Otherwise wouldn't they still need to run it again to know why it failed?

Reply:
I already merged the 1.2 PR before I saw your comment on this one. I'm going to go ahead and merge this one as-is, to keep it common, but we can use an additional PR to change them both to use the more terse option, if desired.